services/drupal/config/sync/metatag.settings.yml

```diff
@@ -0,0 +1,10 @@
+entity_type_groups:
```

Comment:
@pzh7000 We don't really want this file.  This exists because you enabled all of the checkboxes for the tags for this content type, but that's unnecessary.  Let's remove it.